README.md

```diff
@@ -8,8 +8,9 @@ Within this repo, the `/docs/en/` directory is structured as follows:
 
 | Directory             | Description |
 | --------------------- | ----------- |
-| __observability__     | Contains the source files for the [Observability Guide](https://www.elastic.co/guide/en/observability/master/index.html), which includes content for APM, Logs, Metrics, Synthetics, User experience, and Uptime.|
 | __ingest-management__ | Contains the source files for the [Fleet and Elastic Agent Guide](https://www.elastic.co/guide/en/ingest-management/master/index.html).|
+| __integrations__      | Contains the source files for the [Integrations Developer Guide](https://www.elastic.co/guide/en/integrations-developer/master/index.html)
+| __observability__     | Contains the source files for the [Observability Guide](https://www.elastic.co/guide/en/observability/master/index.html), which includes content for APM, Logs, Metrics, Synthetics, User experience, and Uptime.|
```

Comment:
What do you think about linking to current instead of master? I know this file exists in master (main), but I would guess most users who click these links want to see current instead.

Reply:
good idea! Changed.